kml/serdes_gpx.cpp

```diff
@@ -140,6 +142,34 @@ void GpxParser::ParseColor(std::string const & value)
   m_color = kml::ToRGBA(colorBytes[0], colorBytes[1], colorBytes[2], (char)255);
 }
 
+// https://osmand.net/docs/technical/osmand-file-formats/osmand-gpx/ - "#AARRGGBB" or "#RRGGBB"
+void GpxParser::ParseOsmandColor(std::string const & value) {
+  auto const colorBytes = FromHex(value.substr(1, value.size() - 1));
```

Comment:
Is it possible that value.size() can be 0, and `- 1` will lead to overflow?

Reply:
Check for empty string added